src/stravalib/model.py

```diff
@@ -769,23 +788,25 @@ class ActivityPhoto(BaseModel):
     endpoint to retrieve it
     """
 
-    athlete_id: Optional[int] = None
     activity_id: Optional[int] = None
     activity_name: Optional[str] = None
-    ref: Optional[str] = None
-    uid: Optional[str] = None
-    unique_id: Optional[str] = None
+    athlete_id: Optional[int] = None
     caption: Optional[str] = None
-    type: Optional[str] = None
-    uploaded_at: Optional[datetime] = None
     created_at: Optional[datetime] = None
     created_at_local: Optional[datetime] = None
+    default_photo: Optional[bool] = None
     location: Optional[LatLon] = None
-    urls: Optional[dict[str, str]] = None
-    sizes: Optional[dict[str, Sequence[int]]] = None
     post_id: Optional[int] = None
-    default_photo: Optional[bool] = None
+    sizes: Optional[dict[str, Sequence[int]]] = None
     source: Optional[int] = None
+    type: Optional[int] = None
+    unique_id: Optional[str] = None
+    uploaded_at: Optional[datetime] = None
+    urls: Optional[dict[str, AnyHttpUrl]] = None
+
+    # TODO: i don't see these in the actual returns but they are used below
```

Comment:
I didn't see these in the photo return. but haven't looked into how / whether we need them. i just know we use them below in our code.